org.gabmus.notorious.json

```diff
@@ -0,0 +1,78 @@
+{
+    "app-id": "org.gabmus.notorious",
+    "command": "notorious",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.36",
+    "sdk": "org.gnome.Sdk",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=home:rw"
+    ],
+    "modules": [
+        {
+            "name": "libhandy",
+            "buildsystem": "meson",
+            "config-opts": [
+                "--buildtype=release"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://source.puri.sm/Librem5/libhandy",
+                    "tag": "v0.0.13"
+                }
+            ]
+        },
+        {
+            "name": "otf-source-code-pro",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/adobe-fonts/source-code-pro/archive/2.030R-ro/1.050R-it.tar.gz",
+                    "sha256": "a4e4dd59b8e0a436b934f0f612c2e91b5932910c6d1c3b7d1a5a9f389c86302b"
+                }
+            ],
+            "build-commands": [
+            	"mkdir -p /app/share/fonts/",
+            	"install -d /app/share/fonts/adobe-source-code-pro/",
+            	"install -t /app/share/fonts/adobe-source-code-pro/ -m644 OTF/*.otf",
+            	"fc-cache -fs"
+            ]
+        },
+        {
+            "name": "gnome-builder-gtksourceview4-colorschemes",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.gnome.org/GNOME/gnome-builder.git",
```

Comment:
Why not just use two sources of type file that points to the two color scheme files?

Reply:
Guess I overthought it. Sure can do.

Reply:
just pushed it, let me rebuild